packages/core/frontend/sirius-components-core/src/monitoring/MonitoringContext.types.ts

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { Span } from './useMonitoring.types';
+
+export interface MonitoringContextContextValue {
+  currentSpan: Span | null;
+  endSpan: () => void;
+  addMeasurements: (id: string, measurement: number, unit: string, ...tags: string[]) => void;
+  startSpan: (id: string) => void;
+}
+
+export interface MonitoringContextContextProviderProps {
+  children: React.ReactNode;
+}
+
+export interface MonitoringContextContextProviderState {
+  currentSpan: Span | null;
+}
```

Comment:
No need for that double context everywhere

Reply:
done